docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -175,130 +168,122 @@ You can create a `kubeconfig` file for previously created `yugabyte-platform-uni
 
-- **Service Account** provide the name of the service account which has necessary access to manage the cluster, refer to [Create cluster](../../../../deploy/kubernetes/single-zone/oss/helm-chart/#create-cluster).
-
-- **Kube Config** there are two ways to specify the kube config for an availability zone.
-  - Specify at **provider level** in the provider form as shown above. If specified, this configuration file will be used for all availability zones in all regions.
-  - Specify at **zone level** inside of the region form as described below, this is especially needed for **multi-az** or **multi-region** deployments.
-
-- **Image Registry** specifies where to pull YugabyteDB image from leave this to default, unless you are hosting the registry on your end.
-
-- **Pull Secret**, the Enterprise YugabyteDB image is in a private repo and you need to upload the pull secret to download the image, your sales representative should have provided this secret.
-
-A filled in form looks something like this:
+The following illustration shows the completed form:
 
 <img title="K8s Configuration -- filled" alt="K8s Configuration -- filled" class="expandable-image" src="/images/ee/k8s-setup/k8s-configure-filled.png" />
 
 ## Configure the region and zones
 
-Click **Add Region** to open the modal.
-
-- Specify a Region and the dialog will expand to show the zone form.
-
-- **Zone**, enter a zone label, keep in mind this label should match with your failure domain zone label `failure-domain.beta.kubernetes.io/zone`
+Continue configuring your Kubernetes provider by clicking **Add Region** and completing the **Add new region** dialog, as follows:
 
-- **Storage Class** is *optional*, it takes a comma delimited value, if not specified would default to standard, please make sure this storage class exists in your k8s cluster.
-
-- **Kube Config** is *optional* if specified at provider level or else `required`
-
-- **Namespace**, *optional* if provided SA have the `Cluster Admin` permissions else `required`. The SA used in provided `kubeconfig` should have access to this namespace.
+- Use the `Region` field to select the region.
+- Use the **Zone** field to select a zone label that should match with your failure domain zone label `failure-domain.beta.kubernetes.io/zone`.
+- Optionally, use the **Storage Class** field to enter a comma-delimited value. If you do not specify this value, it would default to standard. You need to ensure that this storage class exists in your Kubernetes cluster.
+- Use the **Namespace** field to specify the namespace. If provided SA has the `Cluster Admin` permissions, you are not required to complete this field. The SA used in the provided `kubeconfig` file should have access to this namespace.
+- Use **Kube Config** to upload the configuration file. If this file is available at provider level, you are not required to supply it.
```

Comment:
First time 'SA' used, just spell out? eg - If the provided ServiceAccount
The ServiceAccount used

Reply:
Changed to ServiceAccount